packages/examples/project-a/federation.config.js

```diff
@@ -39,9 +39,7 @@ export const federationconfig = (bundler) => ({
       eager: true,
     },
     'react-dom': {},
-    uuid: {
-      import: false,
-    },
```

Comment:
This is done because @module-federation/runtime doesn't support import: false yet.

Bug: https://github.com/module-federation/universe/issues/2020